docs/dev/docker-hub.md

```diff
@@ -151,6 +136,28 @@ The Docker image uses supervisord to manage ScyllaDB processes. You can restart
 docker exec -it some-scylla supervisorctl restart scylla
 ```
 
+### aio-max-nr Value
+
+Before starting the cluster, make sure the [aio-max-nr](https://www.kernel.org/doc/Documentation/sysctl/fs.txt/)  value is high enough (typically 10485760 or higher). This parameter determines the maximum number of allowable Asynchronous non-blocking I/O (AIO) concurrent requests by the Linux Kernel, and it helps ScyllaDB perform in a heavy I/O workload environment.
+If you encounter the error:
+
+```
+FATAL: Exception during startup, aborting: std::runtime_error (Could not setup Async I/O: Resource temporarily unavailable. The most common cause is not enough request capacity in /proc/sys/fs/aio-max-nr. Try increasing that number or reducing the amount of logical CPUs available for your application)
```

Comment:
i think we also print out warning message like
```
Requested AIO slots too large, please increase request capacity in /proc/sys/fs/aio-max-nr. configured:{} available:{} requested:{}
```

the place holders "{}" above are replaced with numbers when the message is formatted by Scylla. so probably we can encourage user to use a number higher than the "requested" number? "1048576" is a safe number, as it should work on a 96-core system with the default settings.